pallets/block-rewards/src/tests.rs

```diff
@@ -5,7 +5,7 @@ use sp_runtime::traits::BadOrigin;
 use super::*;
 use crate::mock::*;
 
-const REWARD: u128 = 100 * crate::mock::ExistentialDeposit::get();
+const REWARD: u128 = 100 + crate::mock::ExistentialDeposit::get();
```

Comment:
Here's the change in case you positive output from https://github.com/centrifuge/centrifuge-chain/pull/1577#discussion_r1372059157